app/Http/Controllers/Secretariat/UserController.php

```
@@ -379,6 +379,50 @@ public function updatePassword(Request $request): RedirectResponse
         return redirect()->back()->with('message', __('general.successful_modification'));
     }
 
+    /**
+     * Revoke a personal access token for the user.
+     * @param Request $request
+     * @param User $user
+     * @param int $token
+     * @return RedirectResponse
+     * @throws AuthorizationException
+     */
+    public function revokeAccessToken(Request $request, User $user, int $token): RedirectResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)